src/splitting-a-field-into-multiple-fields.md

```diff
@@ -10,7 +10,7 @@ https://forums.ankiweb.net/t/cloze-type-to-basic-type/23305/2
 
```

Comment:
As long as you're cleaning up this page, can you figure out a way to make the link above work? It just shows as plain text in the published FAQ.
https://faqs.ankiweb.net/splitting-a-field-into-multiple-fields.html